src/ExRam.Gremlinq.Core/Queries/Interfaces/IGremlinQuery.cs

```diff
@@ -123,6 +123,8 @@ public interface IGremlinQueryBaseRec<TSelf> : IGremlinQueryBase
         TSelf Order(Func<IOrderBuilder<TSelf>, IOrderBuilderWithBy<TSelf>> projection);
         TSelf OrderLocal(Func<IOrderBuilder<TSelf>, IOrderBuilderWithBy<TSelf>> projection);
 
+        TSelf Path();
```

Comment:
But is it? If I have an `IVertexGremlinQuery<Element>` and call Path on it, the resulting query doesn't have vertex semantics, nor is it typed on `Element` (i.e. query results with a Path step are unlikely able to be deserialized to Element). It's rather something like `IValueGremlinQuery<SomePathClass>` for a reasonably choice of SomePathClass. So it must be decided what this SomePathClass class must look like.

Reply:
Thanks. Fixed and tested